handlers/twitter.js

```diff
@@ -1,23 +1,39 @@
-const Twitter = require('twitter')
+const Twitter = require('twit')
 const morph = require('morphmorph')
 
 const RATE_LIMIT_CODE = 420
+const MAX_ALT_TEXT_LENGTH = 420
 
 const client = new Twitter({
   consumer_key: process.env.TWITTER_CONSUMER_KEY,
   consumer_secret: process.env.TWITTER_CONSUMER_SECRET,
-  access_token_key: process.env.TWITTER_ACCESS_TOKEN_KEY,
+  access_token: process.env.TWITTER_ACCESS_TOKEN_KEY,
   access_token_secret: process.env.TWITTER_ACCESS_TOKEN_SECRET
 })
 
 const uploadImage = data => client.post('media/upload', { media_data: data })
-const uploadTweet = (media = {}) =>
-  client.post('statuses/update', {
-    status: `Carbon Copy #${media.media_id_string.slice(0, 8)}`,
-    media_ids: media.media_id_string
-  })
+const uploadMetadata = (altText, twitterRes = {}) => {
+  if (!altText)
+    return twitterRes
 
-const extractImageUrl = morph.get('entities.media.0.display_url')
+  const formattedAltText = altText.length > MAX_ALT_TEXT_LENGTH
```

Comment:
Just FYI (and maybe we want to make this change) but if the length is shorter than the second `slice` param if will just return the whole string. So you don't need the conditional check. 

Are you making the check because you wanted it `MAX_ALT_TEXT_LENGTH - 3`? Because we could probably just do `.slice(0, MAX_ALT_TEXT_LENGTH - 3)` no matter.

Reply:
Yup, but you'd still need a length check for the ellipsis